front/src/modules/ui/layout/right-drawer/components/RightDrawer.tsx

```diff
@@ -1,6 +1,8 @@
+import { useEffect } from 'react';
 import styled from '@emotion/styled';
 import { useRecoilState } from 'recoil';
 
+import { captureHotkeyTypeInFocusState } from '@/hotkeys/states/captureHotkeyTypeInFocusState';
```

Comment:
Is this specific focus behaviour tested ?

Reply:
no, and we are going to remove it completely on Monday as we merge Lucas' refacto :(